Formula/openjdk.rb

```diff
@@ -71,26 +93,50 @@ def install
     # Since openjdk has move their development from mercurial to git and GitHub
     # this approach may need some changes in the future
     #
-    build = File.read(".hg_archival.txt")
-                .scan(/^tag: jdk-#{version}\+(.+)$/)
-                .map(&:first)
-                .map(&:to_i)
-                .max
-    raise "cannot find build number in .hg_archival.txt" if build.nil?
+    if Hardware::CPU.arm?
+      build = File.read(".hgtags")
```

Comment:
I was testing this approach on 15.0.1 (to allow us to clean up the comment above) and it reports as 15.0.1+8 instead of 15.0.1+9.  @VladimirKempik moving forward, should we add `+1` to the last tag in `.hgtags`?  How should we calculate this value moving forward?

Reply:
How much of a deal-breaker is that? Testing has been progressing nicely so far, so I planned to merge this tomorrow. Should I wait for a solution to this, or is it a minor issue?

Reply:
> How much of a deal-breaker is that? Testing has been progressing nicely so far, so I planned to merge this tomorrow. Should I wait for a solution to this, or is it a minor issue?

I was thinking the same thing when I saw the "ready to merge" tag.  I suppose it can be tackled at some future time when this is consolidated, but I'd still like to know moving forward as the comment suggests the old versioning information will eventually go away.